### ISSUE TITLE
Sync interruption due to header difficulty

### ISSUE THREAD

**Issue by aakselrod** [lightninglabs/neutrino#25]
Multiple users are reporting persistent errors similar to the following:

```
lnd: 2017-10-20 03:29:18.639 [INF] BTCN: Syncing to block height 1210641 from peer 165.227.7.29:18333 2017-10-20 03:29:18.639 [INF] BTCN: Downloading headers for blocks 930001 to 1000007 from peer 165.227.7.29:18333
lnd: 2017-10-20 03:29:19.124 [WRN] BTCN: Header doesn't pass sanity check: block target difficulty of 00000000ffff0000000000000000000000000000000000000000000000000000 is higher than max of 0000000000000202f00000000000000000000000000000000000000000000000 -- disconnecting peer 2017-10-20 03:29:19.124 [INF] BTCN: Lost peer 165.227.7.29:18333 (outbound)
lnd: 2017-10-20 03:29:19.124 [WRN] BTCN: No sync peer candidates available
lnd: 2017-10-20 03:29:24.308 [INF] BTCN: New valid peer 165.227.7.29:18333 (outbound) (/btcwire:0.5.0/btcd:0.12.0/)
lnd: 2017-10-20 03:29:24.309 [INF] BTCN: Syncing to block height 1210641 from peer 165.227.7.29:18333 2017-10-20 03:29:24.309 [INF] BTCN: Downloading headers for blocks 930001 to 1000007 from peer 165.227.7.29:18333
lnd: 2017-10-20 03:29:24.813 [WRN] BTCN: Header doesn't pass sanity check: block target difficulty of 00000000ffff0000000000000000000000000000000000000000000000000000 is higher than max of 0000000000000202f00000000000000000000000000000000000000000000000 -- disconnecting peer 2017-10-20 03:29:24.813 [INF] BTCN: Lost peer 165.227.7.29:18333 (outbound)
lnd: 2017-10-20 03:29:24.813 [WRN] BTCN: No sync peer candidates available
```

**Comment by aakselrod** [comment]
I don't seem to be able to reproduce this error. Is there a chance someone could attach a copy of their neutrino files from a node that is experiencing it? I'll need copies of `data/testnet3/bitcoin/{*.bin,neutrino.db}`. Please attach to this issue.

**Comment by aakselrod** [comment]
I have a copy of the relevant files that let me reproduce the error.

**Comment by aakselrod** [comment]
I am now able to reproduce the issue and a fix is forthcoming in #27.

**Comment by Roasbeef** [comment]
Fixed by #27 

Closing.
